ldap/data_source_entries.go

```diff
@@ -74,6 +76,24 @@ func dataSourceLDAPEntries() *schema.Resource {
 				Optional:    true,
 				Elem:        &schema.Schema{Type: schema.TypeString},
 			},
+			attributeNamePagingSize: {
+				Description: "Desired page size for the search request. Use -1 to retrieve all results without pagination, or a value greater than 0 to enable paginated queries. Defaults to 100.",
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Default:     100,
+				ValidateDiagFunc: func(val interface{}, path cty.Path) diag.Diagnostics {
```

Comment:
pleas use  https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2@v2.19.0/helper/validation#IntBetween

Reply:
I considered it, but since the value 0 is invalid I opted for this approach, hope that makes sense.

Reply:
I suppose if we go with `0` as "no limit" we can use https://github.com/hashicorp/terraform-plugin-sdk/blob/v2.19.0/helper/validation/int.go#L31 with `min` == 0

Reply:
updated code to have `0` mean "no limit" and using `IntAtLeast` for input validation